rallyrolebot/cogs/defaults_cog.py

```diff
@@ -37,6 +35,115 @@ async def cog_command_error(self, ctx, error):
             type(error), error, error.__traceback__, file=sys.stderr
         )
 
+    @staticmethod
+    async def update_setting(ctx, alert, alert_nr, value, setting):
+        # check if any value is missing
+        if not alert or not alert_nr or not value:
```

Comment:
This check may not be necessary as there's already an error implementation for missing arguments